readmore.md

```diff
@@ -0,0 +1,186 @@
+# MuSig2 Implementation with libsecp256k1
+
+This is a MuSig2 implementation using the libsecp256k1 library for EC operations.
+
+To install the library, follow the directions in [libsecp256k1](https://github.com/bitcoin-core/secp256k1).
+The library provides an optimized C library for ECDSA signatures and secret and public key operations on curve ecp256k1, as well as usage examples including ECDSA and Schnorr signatures.
+
+This implementation requires configuring the libsecp256k1 with an additional flag `--enable-module-schnorrsig` as stated in [libsecp256k1](https://github.com/bitcoin-core/secp256k1).
+
+Run the example with examplemusig2.c
+
+```
+make ctest
+./ctest_run/ctest
+```
+
+Run the Google tests with
+
+```
+make gtest
+./gtest_run/gtest
+```
+
+## Schnorr Signature of libsecp256k1
+
+The library offers an optional module implementing Schnorr signatures over the curve `secp256k1`.
+The scheme is compatible with [BIP-340](https://github.com/bitcoin/bips/blob/master/bip-0340.mediawiki) that produces 64-byte Schnorr signatures.
+BIP-340 standardized the scheme with some modifications for practical purposes.
+
+### Schnorr Signature in a nutshell
+
+Let $p$ be a prime and $E$ be an elliptic curve defined over the finite field $F_p$ with the base point $G$.
```

Comment:
You probably want to specify what the role of $p$ is (i.e. that it is the order of the base point $G$).